packages/rum/test/rum.spec.ts

```diff
@@ -261,6 +262,7 @@ describe('rum session', () => {
   })
 
   afterEach(() => {
+    viewCollection.stop()
```

Comment:
viewCollection may not be defined here, because all test cases aren't defining it. Maybe test if it is defined before using it?

Reply:
Currently, all tests are defining it but you are right, it could not always be the case.
I would be in favor of trying to move the startViewCollection in the beforeEach if it is possible, it could factorize some test setup logic.
wdyt?